frontend/public/locales/zh/nav.json

```diff
@@ -0,0 +1,68 @@
+{
+  "Monitoring": "监控",
+  "Alerting": "警报",
+  "Metrics": "指标",
+  "Dashboards": "仪表板",
+  "Home": "家",
+  "Overview": "概述",
+  "Projects": "项目",
+  "Search": "搜索",
+  "Explore": "探索",
+  "Events": "事件",
+  "Operators": "Operator",
+  "Workloads": "工作负载",
+  "Pods": "Pod",
+  "Deployments": "部署",
+  "Deployment Configs": "部署配置",
+  "Stateful Sets": "有状态集",
+  "Secrets": "Secrets",
+  "Config Maps": "Config Map",
+  "WorkloadsSeparator": "WorkloadsSeparator",
+  "Cron Jobs": "Cron Jobs",
+  "Jobs": "作业",
+  "Daemon Sets": "守护进程集",
+  "Replica Sets": "副本集",
+  "Replication Controllers": "复制控制器",
+  "Horizontal Pod Autoscalers": "横向自动扩展",
+  "Serverless": "Serverless",
+  "Networking": "网络",
+  "Services": "服务",
+  "Routes": "路由",
+  "Ingresses": "入站",
+  "Network Policies": "网络政策",
+  "Storage": "存储",
+  "Persistent Volumes": "持久性卷（PV）",
+  "Persistent Volume Claims": "持久性卷声明 (PVC)",
+  "Storage Classes": "存储类",
+  "Volume Snapshots": "卷快照",
+  "Volume Snapshot Classes": "卷快照类",
+  "Builds": "Builds",
+  "Build Configs": "构建配置",
+  "Image Streams": "镜像流",
+  "Pipelines": "Pipelines",
+  "Service Catalog": "Service Catalog",
+  "Provisioned Services": "置备的服务",
+  "Broker Management": "代理管理",
+  "Compute": "Compute",
+  "Nodes": "节点",
+  "Machines": "Machine",
+  "Machine Sets": "机器集",
+  "Machine Autoscalers": "机器自动扩展",
+  "Machine Health Checks": "机器健康状态检查",
+  "ComputeSeparator": "ComputeSeparator",
+  "Machine Configs": "机器配置",
+  "Machine Config Pools": "机器配置池",
+  "User Management": "用户管理",
+  "Users": "用户",
+  "Groups": "组",
+  "Service Accounts": "服务帐户",
+  "Roles": "角色",
+  "Role Bindings": "角色绑定",
+  "Administration": "管理",
+  "Cluster Settings": "集群设置",
+  "Namespaces": "命名空间",
+  "Resource Quotas": "资源配额",
+  "Limit Ranges": "限制范围",
+  "Chargeback": "Chargeback",
+  "Custom Resource Definitions": "自定义资源定义"
+}
```

Comment:
Some fields in nav.json are not consistently translated into Japanese and Chinese:  "Ingresses","Horizontal Pod Autoscalers","Compute","Machine","Chargeback","Service Catalog". I'm not sure when they need be translated.

Reply:
They added translations for Ingresses and Horizontal Pod Autoscalers in the latest revision.

Reply:
nav file does not exist for Japanese in this drop so we'll need this file in order to address to the comment raised, also tagging @rh-tfu to clarify this

Reply:
@yanpzhan, do you have suggestion for the Chinese term of "Chargeback" or we can leave it in English?

Reply:
I think we can leave "Compute" and "Service Catalog" in English

Reply:
@rh-tfu I didn't get a proper Chinese term for "Chargeback", It's ok for me to leave it in English.